src/index.html

```diff
@@ -11,7 +11,158 @@
+          Name:
+          <input type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            class="year"
+            type="number"
+            name="year"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="checkbox ">
+        </label>
+      </fieldset>
+
+      <!--Registration-->
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <!--An interesting fact about you!-->
+      <fieldset class="fieldset">
+        <legend class="legend">An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="radio"
+            value="Yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="radio"
+            value="No"
+          >
+          No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            class="time"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+          name="car"
+          required
+          multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <!--Additional info-->
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="text"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
Fix it due to checklist
![image](https://user-images.githubusercontent.com/80700247/230766630-c0d6c3c4-fee6-4d14-85cf-cd556abf4a62.png)